definition/data/sheets/AuthorisationCaseState.json

```diff
@@ -12,5 +12,15 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "readyForDirections", "UserRole": "citizen", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "stayed", "UserRole": "citizen", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "open", "UserRole": "letter-holder", "CRUD": "CRUD"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "create", "UserRole": "letter-holder", "CRUD": "CRUD"}
+  {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "create", "UserRole": "letter-holder", "CRUD": "CRUD"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "orderForJudgeReview", "UserRole": "caseworker-cmc-legaladvisor", "CRUD": "R"},
```

Comment:
Please add 'R' for orderForJudgeReview and orderForLAReview  for caseworker-cmc

Reply:
👍